ocaml/xapi/xapi_host.ml

```diff
@@ -2725,7 +2725,7 @@ let really_write_uefi_certificates_to_disk ~__context ~host:_ ~value =
                  [Unix.O_WRONLY; Unix.O_CREAT; Unix.O_TRUNC]
                  0o644
              in
-             debug "override_uefi_certs: copy_file %s->%s" src dst ;
```

Comment:
these names are outdated, using `debug "%s: ..." __FUNCTION__ ...` would be better